packages/oauth/src/stores/file-store.spec.js

```diff
@@ -92,24 +98,36 @@ describe('FileInstallationStore', async () => {
 
   it('should store the latest installation', async () => {
     const installationStore = new FileInstallationStore({ baseDir: os.tmpdir() });
-    const installer = new InstallProvider({ clientId, clientSecret, stateSecret, installationStore });
```

Comment:
Except for the `authorize` tests, we don't need to instantiate `InstallProvider`.  In fact, this test uses only `installer.installationStore`.